package.xml

```diff
@@ -52,8 +52,25 @@ https://github.com/php/pecl-search_engine-solr
   <!-- Notes for the latest release -->
   <notes>
 		<![CDATA[
- - PHP 8.1 compatible
- - PHP 8.2 compatible
+Features:
+- Support Highlight Query (#28)
+- Support PHP-8.3
```

Comment:
@omars44 since we are removing support for <=7.3, we probably want to mention it here in these notes (and in the notes below also)?

Reply:
sure